core/services/src/main/java/org/exoplatform/services/cms/clouddrives/DisconnectCloudDriveServiceImpl.java

```diff
@@ -0,0 +1,91 @@
+package org.exoplatform.services.cms.clouddrives;
```

Comment:
IMO you don't need an additionnal service file, you can use [CloudDriveServiceImpl.java](https://github.com/exoplatform/ecms/blob/task/59140/core/services/src/main/java/org/exoplatform/services/cms/clouddrives/CloudDriveServiceImpl.java)